### PR TITLE
Update signing to v2

### PR DESCRIPTION
**Pull request by mattleibow**
"Upgrade" the signing process to the new, microbuild version.

### CODE REVIEW

File: SignList.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <FirstParty Include="Xamarin.Firebase.*.dll" />
+    <FirstParty Include="Xamarin.Google.*.dll" />
```
Comment (mattleibow) [inline SignList.xml:5]: Are there other prefixes that we want to sign?

Comment (Redth) [reply SignList.xml:5]: Seems like the actual .dll patterns are:

 - `Firebase.*.dll`
 - `Google.*.dll`

https://github.com/xamarin/GoogleApisForiOSComponents/search?q=%3CAssemblyName%3E

File: SignList.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <FirstParty Include="Xamarin.Firebase.*.dll" />
```
Comment (Redth) [inline SignList.xml:4]:
```suggestion
    <FirstParty Include="Firebase.*.dll" />
```

File: SignList.xml (hunk 1 above)
Comment (Redth) [inline SignList.xml:5]:
```suggestion
    <FirstParty Include="Google.*.dll" />
```

**Review by Redth — APPROVED** [review]

**Comment by mattleibow** [comment]
Let's try out that new signing code!

**Pull request merged**